src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,137 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75F;
+    static final int INITIAL_THRESHOLD = 12;
+    private MyHashMap.Node<K, V>[] table;
```

Comment:
```suggestion
    private Node<K, V>[] table;
```
можна одразу так тип вказувати

Reply:
Всюди видаляв, виявилось - не всюди :-)